rustfmt.toml

```diff
@@ -1 +1,11 @@
-ignore = ["src/json"]
```

Comment:
Why did we remove this ignore?

Reply:
Good catch. I had this on my list because we are currently getting this error on `main`:

```
Warning: can't set `ignore = IgnoreList { path_set: {"src/json"}, rustfmt_toml_path: "" }`, unstable features are only available in nightly channel.
```

I will double check if the rule/path is still valid.

Reply:
Ok this was ignoring the files under `src/json` but this folder no longer exists. Was there any `.rs` file under this folder that was supposed to be ignored by `rustfmt`? 🤔

Reply:
This was added to ignore `json` files used for tests, although nowadays they are scattered across tests modules. Maybe we should think about a more ad-hoc solution to flagging modules to be ignored by `rustfmt`. Or we may just as well want rustfmt to format those, if it is fact doing that.

Reply:
> Or we may just as well want rustfmt to format those, if it is fact doing that.

I could not find any information regarding `rustfmt` formatting `json` files (I would say that would be a bit weird too 😅 )